servers/republik/graphql/schema-types.js

```diff
@@ -54,7 +54,8 @@ extend type User {
   pgpPublicKeyId: String
   emailAccessRole: AccessRole
 
-  biography: String
+  biography: String @deprecated(reason: "use biographyMdast instead")
```

Comment:
```suggestion
  # as raw text, use `biographyContent` to get mdast
  biography: String
```

Still needed for editing.